packages/mdc-card/mdc-card.scss

```diff
@@ -24,15 +24,15 @@
 // postcss-bem-linter: define card
 
 .mdc-card {
-  @include mdc-card-fill-color(background);
+  @include mdc-card-fill-color(surface);
   @include mdc-card-corner-radius(2px);
   @include mdc-elevation(2);
   @include mdc-card-container-layout_;
 }
 
 .mdc-card--stroked {
   @include mdc-elevation(0);
-  @include mdc-card-stroke($mdc-card-stroke-color);
+  @include mdc-card-stroke(mix(mdc-theme-prop-value(on-surface), mdc-theme-prop-value(surface), 12%));
```

Comment:
Can we update the variable instead of replacing it here? (IIRC I added that variable for reuse with shape, at least in demos, but potentially of use to users for similar purposes as well)

Also FYI this results in changing from `#dbdbdb` to `#e0e0e0`